roles/openshift_node/defaults/main.yml

```diff
@@ -2,7 +2,8 @@
 r_openshift_node_firewall_enabled: "{{ os_firewall_enabled | default(True) }}"
 r_openshift_node_use_firewalld: "{{ os_firewall_use_firewalld | default(False) }}"
 
-openshift_service_type: "{{ openshift.common.service_type }}"
+openshift_deployment_type: "{{ deployment_type | default('origin') }}"
```

Comment:
why are we using deployment_type here?

Reply:
I think that deployment_type was the original variable to use and is still valid.  I can remove this and depend on openshift_deployment_type.